CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Add specific classes to `filterGroup` and `filterItem` of the `filterAccordion` (filter mobile)
+
```

Comment:
```suggestion
## [Unreleased]
### Added
- Add specific classes to `filterGroup` and `filterItem` of the `filterAccordion` (filter mobile)

```

Reply:
I know it looks weird but removing this line should enable Git to merge this without conflicts :)